tools/topology/sof-icl-rt711-rt1308-rt715-hdmi.m4

```diff
@@ -71,7 +73,7 @@ PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
 
 # Low Latency capture pipeline 5 on PCM 4 using max 2 channels of s32le.
 # Schedule 48 frames per 1000us deadline on core 0 with priority 0
-PIPELINE_PCM_ADD(sof/pipe-highpass-capture.m4,
+PIPELINE_PCM_ADD(sof/pipe-eq-capture.m4,
```

Comment:
this change does not look correct, we recently had @singalsu add a high-pass filter to remove a DC component on startup, why would we want to modify this?

Reply:
I have already seen the discussion, the eq serves as a highpass.

Reply:
@plbossart @paulstelian97 Please check my upper comments. The main purpose is we need a switch controller which enables MIC MUTE LED feature.

Reply:
> ```
> C_CONTROLMIXER(Master Capture Switch, PIPELINE_ID,
>         CONTROLMIXER_OPS(volsw, 259 binds the mixer control to switch get/put handlers, 259, 259),
>         CONTROLMIXER_MAX(max 1 indicates switch type control, 1),
>         false,
>         ,
>         Channel register and shift for Front Left/Right,
>         LIST(`  ', KCONTROL_CHANNEL(FL, 2, 0), KCONTROL_CHANNEL(FR, 2, 1)),
>         "1", "1")
> ```
> 
> This means the switch controller will enable MIC MUTE LED feature.
> This is why we need pipe-eq-capture.m4

@libinyang Sorry, please allow me to ask a basic question. How do we control the MUTE LED and how does the switch enable the MIC MUTE LED feature? Is there a corresponding driver to control the LED?  Can we always enable the MUTE LED feature instead of creating a switch for it?

Reply:
@bardliao There are platform driver, for example, drivers/platform/x86/dell-xxx for dell platforms. In this platform, it will define how LED is controlled and how speical key is treated (To compare, most keys are treated in the keyboard driver). And there is a LED driver for audio, please refer drivers/leds/*. In audio driver, when it need change the LED state, it will call leds driver, which will call dell platform driver. So the sequence is:
key pressed -> (dell) platform/keyboard driver -> userspace -> pulseaudio set the audio mixer -> audio driver kcontrol -> led driver -> (dell) platform driver